### ISSUE TITLE
How can I make a gradient color ?

### ISSUE THREAD

**Issue by wilianos** [spite/THREE.MeshLine#69]
Hi, i am trying to draw a line with a gradient color ? Is there any option ? Thanks.

**Comment by dyh333** [comment]
i want to know it too. does it support ? @spite

**Comment by anyemelody** [comment]
Anyone figure it out? I want to know too!

**Comment by spite** [comment]
Not really, except creating a CanvasTexture from a canvas that has the gradient, and setting that texture as the line texture.

**Comment by trusktr** [comment]
I'm implementing this in my fork. Basically it's a matter of defining a shader `attribute` that contains the colors for the vertices, then passing them to the fragment shader as a `varying`. It's not too hard. Happy to share once I've done it.

**Comment by layonthebeech** [comment]
@trusktr did you ever get around to this?

**Comment by rubychilds** [comment]
@trusktr would be interested here too!
